tests/autorestart_test/test_container_autorestart.py

```diff
@@ -0,0 +1,101 @@
+"""
+Check the auto-restart feature of syncd, swss, teamd, bgp and dhcp_relay
+"""
+import time
+import logging
+
+from common.utilities import wait_until
+
+CONTAINER_STOP_TEST_WAITING_TIME = 30
```

Comment:
Rename to `CONTAINER_STOP_THRESHOLD_SECS`

Reply:
Will update it.